exercises/F-predicates/exercise.js

```diff
@@ -6,10 +6,22 @@
 */
 
 // Finish the predicate function to test if the passed number is negative (less than zero)
-function isNegative(number) {}
+function isNegative(number) {
+  if (number < 0) {
+    return true;
+  } else {
+    return false;
+  }
```

Comment:
again, something to think about - could this be done in 1 line? (when returning true or false)